### ISSUE TITLE
please add the ability to zoom in and out

### ISSUE THREAD

**Issue by ginobean** [darkdukey/Cocos-Studio#47]
Take a look at SpriteBuilder. You can use Command + and Command - to zoom in out of the design view. Please add something similar for Cocos Studio.

As it is, I can't see what my design looks like until I actually publish and then run it.

**Comment by darkdukey** [comment]
I think it's already supported there is a zoom control on bottom right corner

**Comment by ginobean** [comment]
I see it now, but it still seems a bit clumsy to use.

Take a look at how SpriteBuilder does it. You can use Command-"-" and Command-"+" to zoom in and out easily.  and then, to reset the view, Command-"0". 

Basically, I'm guessing all of the code to handle zooming in and out is there, so all you have to do is to add handling for zoom-in and zoom-out in the style of SpriteBuilder. Zooming granularity is better in SpriteBuilder as well.

**Comment by darkdukey** [comment]
Yes, I agree, I'll update this if there is improvements in the upcoming release.
